alexthompson207/current-climate
# Test/main page

# PR CurrentClimate

### What does this do?


### Are there any known bugs?


### Summary of the changes
Write happy and sad path tests for homepage view and single article page view.

### What ticket(s) do the changes resolve?

Please Link Issues to this PR as necessary.

### Notes for the Reviewer
All Tests should pass.


### How to Test Changes?
clone or fetch this branch. Run `npm cypress open` all tests in `mainPage.js` and `singleArticleView.js` should pass.

### Next Iterations
- Maybe add a favorites view?
